hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/domain/ArticleRepositoryCustom.kt

```diff
@@ -1,8 +1,12 @@
 package com.devfloor.hongit.api.article.domain
 
 import com.devfloor.hongit.core.article.domain.Article
+import com.devfloor.hongit.core.board.domain.Board
+import org.springframework.data.domain.PageRequest
 
 interface ArticleRepositoryCustom {
     fun findByFavoriteTopFive(): List<Article>
     fun findByViewCountTopFive(): List<Article>
+    fun findAllByBoardOrderByViewCount(board: Board, pageRequest: PageRequest): List<Article>
+    fun findAllByBoardOrderByFavorite(board: Board, pageRequest: PageRequest): List<Article>
 }
```

Comment:
```suggestion
interface ArticleRepositoryCustom {
    fun findByFavoriteTopFive(): List<Article>
    
    fun findByViewCountTopFive(): List<Article>
    
    fun findAllByBoardOrderByViewCount(board: Board, pageRequest: PageRequest): List<Article>
    
    fun findAllByBoardOrderByFavorite(board: Board, pageRequest: PageRequest): List<Article>
}
```

이것도 개인적인 취향인건지 잘 모르겠는데 인터페이스이다보니 코드의 선언이 더 잘 보이게 사이에 공백을 주는게 더 좋아보이긴 하는데 어떠신지???